### PR TITLE
New methods added

### PR DESCRIPTION
**Pull request by ecaliskanunal**

### CODE REVIEW

**Review by GokceKochan — APPROVED** [review]
good

**Review by AndrianHrisotimos — APPROVED** [review]
Done

**Review by Zorlu-Test — APPROVED** [review]
I think I cann

File: src/test/java/Group14.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 public class Group14 {
     public static void main(String[] args) {
         System.out.println("Hello Group 14!");
+        System.out.println("Let's merge!");
     }
 }
```
Comment (fatihcam46) [inline src/test/java/Group14.java:6]: very good

**Comment by AlbanBujar** [comment]
It is good

**Comment by fatihcam46** [comment]
hello

**Pull request merged**